testing/tests/DevExpress.ui.widgets.dataGrid/dataGrid.tests.js

```diff
@@ -3849,6 +3849,37 @@ QUnit.test('column headers visibility when hide removing row in batch editing mo
     assert.strictEqual($dataGrid.find('.dx-datagrid-headers').css('paddingRight'), '0px', 'no headers right padding');
 });
 
+// T852171
+QUnit.test('numberbox input right and left paddings should be equal if spin buttons are showed', function(assert) {
+    // arrange
+    const $dataGrid = $('#dataGrid').dxDataGrid({
+        dataSource: [{ field: 30 }],
+        loadingTimeout: undefined,
+        editing: {
+            mode: 'cell',
+            allowUpdating: true
+        },
+        columns: [{
+            dataField: 'field',
+            dataType: 'number',
+            editorOptions: {
+                showSpinButtons: true
+            }
+        }]
+    });
+
+    const dataGrid = $dataGrid.dxDataGrid('instance');
+
+    // act
+    dataGrid.editCell(0, 0);
+
+    const input = $($dataGrid.find('.dx-editor-cell').find('input')).eq(1);
```

Comment:
1) input -> $input
2) Use ".dx-texteditor-input" instead eq(1)
3) Also check that `$input.length` equal 1 instead of `assert.ok(input)`